Client/Frontend/Browser/Tab.swift

```diff
@@ -195,6 +195,11 @@ class Tab: NSObject {
             configuration!.preferences = WKPreferences()
             configuration!.preferences.javaScriptCanOpenWindowsAutomatically = false
             configuration!.allowsInlineMediaPlayback = true
+            if #available(iOS 13.0, *) {
+                configuration!.defaultWebpagePreferences = WKWebpagePreferences().then {
+                    $0.preferredContentMode = Preferences.General.alwaysRequestDesktopSite.value ? .desktop : .mobile
```

Comment:
Not a big fan of the duplication:

https://github.com/brave/brave-ios/pull/1368/files#diff-c0ee444fbdc7d184b084fb4cc3e534fcR72

But may not easy way around this?